packages/svelte/src/lib/component/Dropzone.svelte

```diff
@@ -0,0 +1,338 @@
+<script lang="ts">
+  import { fromEvent } from "file-selector";
+  import { createEventDispatcher, onDestroy } from "svelte";
+
+  import {
+    fileAccepted,
+    fileMatchSize,
+    isEvtWithFiles,
+    isIeOrEdge,
+    isPropagationStopped,
+    normalizeAccept,
+    onDocumentDragOver,
+    TOO_MANY_FILES_REJECTION,
+    type ComposeFunction,
+    type FileAccept,
+    type FileRejectReason,
+    type InputFile,
+  } from "../utils/dropzone";
+
+  interface Dispatch {
+    drop: {
+      acceptedFiles: any[];
+      rejectReasons: FileRejectReason[];
+      event: Event;
+    };
+    dragEnter: Event;
+    dragLeave: Event;
+    dragOver: Event;
+    dropAccepted: { acceptedFiles: InputFile[]; event: Event };
+    dropRejected: { rejectReasons: FileRejectReason[]; event: Event };
+    fileDialogCancel: void;
+  }
+
+  // default props
+  export let disabled = false;
```

Comment:
Are these intentionally exported? Could they be an object instead of a doze exports?

Reply:
Sure, I'll make them an object